packages/plugin-commands-env/src/node.ts

```diff
@@ -115,14 +120,14 @@ async function downloadAndUnpackZip (
   await fs.promises.unlink(tmp)
 }
 
-function getNodeJSTarball (nodeVersion: string, releaseDir: string) {
+function getNodeJSTarball (nodeVersion: string, releaseDir: string, nodeMirror: string) {
   const platform = process.platform === 'win32' ? 'win' : process.platform
   const arch = normalizeArch(process.platform, process.arch)
   const extension = platform === 'win' ? 'zip' : 'tar.gz'
   const pkgName = `node-v${nodeVersion}-${platform}-${arch}`
   return {
     pkgName,
-    tarball: `https://nodejs.org/download/${releaseDir}/v${nodeVersion}/${pkgName}.${extension}`,
+    tarball: `${nodeMirror}/${releaseDir}/v${nodeVersion}/${pkgName}.${extension}`,
```

Comment:
Do we want to validate the URI in some way? I think someone could set this to a `file:///` URI in theory. That seems fine to me, but maybe there's security ramifications I'm forgetting.

Reply:
I don't think there's a need to validate the protocol